config/install/field.field.localgov_alert_banner.localgov_alert_banner.field_localgov_link.yml

```diff
@@ -0,0 +1,22 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.localgov_alert_banner.field_localgov_link
```

Comment:
So continuing to compare https://github.com/localgovdrupal/localgov_services/blob/b8d65a1a361a05ec368f3156f7d818c12bc93fff/modules/localgov_services_page/config/install/field.field.node.localgov_services_page.localgov_services_parent.yml#L5

The storage, which in the case of services parent is shared between multiple services node bundles so is in the localgov_services_navigation module, is name `localgov_services_parent` and entity `node`.